spec/support/matchers.rb

```diff
@@ -48,6 +48,16 @@
       end
     end
 
+    doc.css("a[href^='https://secure.login.gov']").each do |a|
+      next if a[:lang]
```

Comment:
What's this meant to be for? (I'm guessing it's copied from the block above, but I still don't know its purpose)

Reply:
if a link had an explicit `<a lang="">` attribute then we wouldn't need to worry about the `href` matching the implied locale because the link explicitly targets one

Reply:
Oh, I see we used this to get around the language picker. I'm not sure we need it for https://secure.login.gov sites though. It also feels like [`hreflang`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#hreflang) might be something we'd want to use / key off of instead.